Automattic/newspack-newsletters
# Add all accepted keys to get_post_type_labels()

### All Submissions:






### Changes proposed in this Pull Request:

When registering the CPT we currently don't translate all the labels which means in the editor we might see things like "Post updated." or "Post published.". This PR fixes it 🙂

### How to test the changes in this Pull Request:

1. Go to the editor and send a new newsletter
2. Notice the snackbar
3. Switch to this branch
4. Repeat 1 and 2

### Other information:
